Build/libHttpClient.gdk.bwoi.props

```diff
@@ -82,26 +82,30 @@
   ****************************************************************************************************
   -->
   <PropertyGroup Condition="'$(GDKUseBWOI)' == 'true'">
-    <_AlternativeVCTargetsPath160>$(GDKMSBuildForVS2019)v160\</_AlternativeVCTargetsPath160>
-    <_AlternativeVCTargetsPath150>$(GDKMSBuildForVS2019)v150\</_AlternativeVCTargetsPath150>
+    <_AlternativeVCTargetsPath170>$(GDKMSBuildForVS2022)v170\</_AlternativeVCTargetsPath170>
```

Comment:
Have you concerned that this doesn't break anything for XSAPI/LHC GDK builds? I though at least XSAPI built targeting GDK  bwoi using VS2022 already?

Reply:
@jasonsandlin did you create this file initially?  I thought this was something that came from ChuckW and I'm a little hesitant to update it

Reply:
https://microsoft.visualstudio.com/Xbox.Services/_build/results?buildId=77675685&view=results
Looks like LHC are fine at least. I just added to this property the condition to define these properties only when '$(VisualStudioVersion)' == '17.0' just in case.

I'll test XSAPI pipelines targetting these changes in the meantime.

Reply:
https://microsoft.visualstudio.com/Xbox.Services/_build/results?buildId=77729674&view=results
XSAPI pipelines seem to be running fine too!

Reply:
Here's the section of GDK Build With / Out Install docs that mentions the VCTargetsPath17 property override:
https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/tools-console/usingwithoutinstall/project-configuration-withoutinstall#visual-studio-2022-platform-overrides

Reply:
Gotcha. I forgot that we don't actually build 2022 in XSAPI at all.  I read through the docs you linked and this looks fine to me, assuming it solves the issues you've been having :)